Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,9 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
```

Comment:
All answers are correct, however, you missed the answer for the exercise 2.

Reply:
ex 2 complete